src/Miunie.ConsoleApp/Resources/LanguageResourceCollection/Lang/PhrasesEn.json

```diff
@@ -117,6 +117,24 @@
             "Here it is! https://github.com/discord-bot-tutorial/Miunie",
             "Sure thing, https://github.com/discord-bot-tutorial/Miunie"
         ]
+    },
+    {
+        "Key": "USER_REP_LOG_ENTRY",
+        "Pool": [
+          "{0}Rep received from {1} at {2}. Reason: {3}"
+        ]
+    },
+    {
+      "Key": "USER_REP_LOG_SHOW",
+      "Pool" : [
+        "Latest 5 reputation interactions for {0}:\n---------------------------------------"
+      ]
+    },
+    {
+      "Key": "USER_REP_LOG_SHOW_EMPTY",
+      "Pool" : [
+        "{0} haven't reputation interactions yet!"
```

Comment:
`"{0} don't have any reputation interactions yet!"`
This is correcter English :)

Some extra suggestions for a bigger pool:
`"{0} has never interacted with coolness :smirk:"`
`It's time for {0} to come out of his cave, he never interacted with reputations.`